__tests__/app.test.js

```diff
@@ -482,6 +452,103 @@ describe('GET /api/users', () => {
     })
 })
 
+describe('QUERIES /api/reviews', () => {
+    test('200: Responds with reviews only of category queried', () => {
+        return request(app)
+        .get('/api/reviews?category=social_deduction')
```

Comment:
we can also add the default queries here too! So to check that sort by defaults to date and order by desc.